app/serializers/chat_channel_serializer.rb

```diff
@@ -50,11 +50,11 @@ def chatable
   end
 
   def archive
-    object.chat_channel_archive
+    @archive ||= object.chat_channel_archive
```

Comment:
The memoization here is not required since AR record memoizes the association internally once loaded.

Reply:
Yeah good point, wasn't totally sure so I threw it in anyway, will remove